.gitignore

```diff
@@ -13,21 +13,24 @@ __pycache__
 DartConfiguration.tcl
 .DS_Store
 
-##build
+## Python build directories & artifacts
 htmlcov
 dist/
 cudf.egg-info/
 python/build
 python/cudf/bindings/*.cpp
+build/
+cpp/build/
```

Comment:
This is not a python build directory or artifact. New heading?

Reply:
Oh I see, it's repeated below. I think you can remove this line.

Reply:
Ok. Will do that and update the changelog.